crates/rpc/src/v02/types.rs

```diff
@@ -134,10 +160,22 @@ pub mod request {
         serde(untagged)
     )]
     pub enum BroadcastedInvokeTransaction {
```

Comment:
Why keep `enum` with single variant? :)

Without enum I imagine some utility methods won't be necessary any more as well.

Reply:
Two reasons: I was being lazy, and the second reason is that I'm anticipating an invoke v2 soon-ish so reworking into a struct does not make sense.